backend/src/main/java/org/eclipse/tractusx/puris/backend/common/util/VariablesService.java

```diff
@@ -54,59 +55,41 @@ public class VariablesService {
      * The edrEndpoint to be used during consumer pull asset transfers.
      */
     private String edrEndpoint;
+
     @Value("${puris.edr.deletiontimer}")
     /**
      * The number of minutes before received authentication data
      * in the context of a consumer pull is removed from memory
      */
     private long edrTokenDeletionTimer;
+
     @Value("${puris.baseurl}" + "catena/item-stock/request")
     /**
      * The url under which this application's request endpoint can
      * be reached by external machines.
      */
     private String requestServerEndpoint;
+
```

Comment:
We should reframe the requestAsset and requestServerEndpoint to itemStockSubmodelAssetId and itemStockSubmodelEndpoint.

Please also consider changes in application.properties, docker & helm.